src/features/shared/get-contactmomenten-service.ts

```diff
@@ -122,5 +124,59 @@ export function useContactmomentenByKlantId(
     url.searchParams.set("klant.id", id.value);
     return url.toString();
   }
-  return ServiceResult.fromFetcher(getUrl, fetchContactmomenten);
+  return ServiceResult.fromFetcher(getUrl, fetchContactmomenten, {
+    getUniqueId() {
+      return getUrl() + "contactmoment";
+    },
+  });
+}
+
+export function useContactverzoekenByKlantId(
+  id: Ref<string>,
+  page: Ref<number>
+) {
+  function getUrl() {
+    const url = getAllContactmomentenUrl(page.value);
+    url.searchParams.set("klant.id", id.value);
+    return url.toString();
+  }
+
+  return ServiceResult.fromFetcher(getUrl, fetchContactverzoeken, {
+    getUniqueId() {
+      return getUrl() + "contactverzoek";
+    },
+  });
+}
+
+function fetchContactverzoeken(url: string): Promise<Paginated<any>> {
+  return fetchLoggedIn(url)
+    .then(throwIfNotOk)
+    .then((r) => r.json())
+    .then((x) => parsePagination(x, mapContactverzoekDetail))
+    .then((paginated) => {
+      const page = paginated.page.filter((p) => p !== undefined);
+
+      return { ...paginated, page };
+    });
 }
+
+const mapContactverzoekDetail = (
+  rawContactverzoek: any
+): ContactverzoekDetail | undefined => {
+  if (!rawContactverzoek.embedded.contactmoment.todo) return;
+
+  const contactmoment = rawContactverzoek.embedded.contactmoment;
+  const todo = rawContactverzoek.embedded.contactmoment.embedded.todo;
+
+  return {
+    id: contactmoment.id,
+    datum: formatDateOnly(new Date(contactmoment.registratiedatum)),
+    status: todo.completed ? "Gesloten" : "Open",
+    behandelaar: todo.attendees?.[0] ?? "-",
+    afgerond: todo.completed ? "Ja" : "Nee",
```

Comment:
afgerond en status zijn eigenlijk hetzelfde. is besproken of er een weg kan?

Reply:
Zijn inderdaad hetzelfde, wens is expliciet benoemd om 'm tóch zo te implementeren.